src/Umbraco.Web.UI.Client/src/common/services/blockeditormodelobject.service.js

```diff
@@ -641,7 +641,7 @@
                 };
 
                 // first time instant update of label.
-                blockObject.label = (blockObject.config.label || blockObject.content?.contentTypeName) ?? "" ;
+              blockObject.label = (blockObject.labelInterpolator(blockObject.data) || blockObject.content?.contentTypeName) ?? "";
```

Comment:
Hey there @dKumarmj! Thanks for the help here but unfortunately this seems to break the block grid editor (BGE) completely. 

You're trying to use a deprecated function here: 

![image](https://user-images.githubusercontent.com/304656/233985015-9c7f5ca7-0523-4db7-bcd4-61ef3e36ce3b.png)

And when running the backoffice now, the BGE editor crashes with an error: 

```
TypeError: blockObject.labelInterpolator is not a function
    at BlockEditorModelObject.getBlockObject (umbraco.services.min.js?d=638179324600000000:1:159010)
    at getBlockObject (umbraco.directives.min.js?d=638179324600000000:1:531300)
    at initializeLayoutEntry (umbraco.directives.min.js?d=638179324600000000:1:533194)
    at umbraco.directives.min.js?d=638179324600000000:1:530753
    at Array.forEach (<anonymous>)
    at initializeLayout (umbraco.directives.min.js?d=638179324600000000:1:530721)
    at onLoaded (umbraco.directives.min.js?d=638179324600000000:1:530223)
    at angular.js:18075:37
    at m.$digest (angular.js:19242:15)
    at m.$apply (angular.js:19630:13) 'Possibly unhandled rejection: {}'
```

This also stopped the automated tests from running as you can see in the checks that were not successful. 

Not sure what the fix is but the team said that the call to `$interpolator` is very expensive so that's not the solution here. Could you have another look to see if there's a good alternative?

Reply:
@nul800sebastiaan 

Another alternative is to use the ng-bind-html directive, which allows to bind HTML content to an element and interpolate it using the {{ }} syntax. This can be useful if we need to display HTML content that includes interpolated values. 

It's worth noting that both ng-bind-template and ng-bind-html are less powerful than $interpolate and may not support all the features(custom contexts,only support simple expressions,do not support filters etc..). However, they are generally faster and more efficient, so they may be a good choice if performance is a concern.

it may be a weird thought :) , but can we simply check whether the label startwith (`{{`) ?